pkg/project/controller/controller.go

```diff
@@ -54,48 +54,75 @@ func (c *NamespaceController) Handle(namespace *kapi.Namespace) (err error) {
 }
 
 // deleteAllContent will purge all content in openshift in the specified namespace
-func deleteAllContent(client osclient.Interface, namespace string) (err error) {
-	err = deleteBuildConfigs(client, namespace)
+func deleteAllContent(oc osclient.Interface, kc kclient.Interface, namespace string) (err error) {
+	err = deleteBuildConfigs(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteBuilds(client, namespace)
+	err = deleteBuilds(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteDeploymentConfigs(client, namespace)
+	err = deleteDeploymentConfigs(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteImageStreams(client, namespace)
+	err = deleteImageStreams(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deletePolicies(client, namespace)
+	err = deletePolicies(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deletePolicyBindings(client, namespace)
+	err = deletePolicyBindings(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteRoleBindings(client, namespace)
+	err = deleteRoleBindings(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteRoles(client, namespace)
+	err = deleteRoles(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteRoutes(client, namespace)
+	err = deleteRoutes(oc, namespace)
 	if err != nil {
 		return err
 	}
-	err = deleteTemplates(client, namespace)
+	err = deleteTemplates(oc, namespace)
 	if err != nil {
 		return err
 	}
-	return nil
+	return deleteServiceAccounts(kc, namespace)
+}
+
+func deleteServiceAccounts(client kclient.ServiceAccountsNamespacer, ns string) error {
```

Comment:
the kubernetes namespace controller already cleans up service accounts... we should only be cleaning up origin objects